_sass/layout/maintenance.scss

```diff
@@ -70,6 +70,10 @@
             margin-right: 10px;
             margin-bottom: 8px;
 
+            img.small-logo {
```

Comment:
We should give a more "semantic" name to this class, that is more about they *why* it's special than *how* it's special. Consider calling it `logo-with-wide-aspect-ratio`, for example.